AUTHORS

```diff
@@ -1,23 +1,21 @@
-Sphinx is written and maintained by Georg Brandl <georg@python.org>.
+Sphinx's maintainers, listed alphabetically by forename, are:
 
-Substantial parts of the templates were written by Armin Ronacher
-<armin.ronacher@active-4.com>.
-
-Other co-maintainers:
-
-* Takayuki Shimizukawa <shimizukawa@gmail.com>
+* Adam Turner <@AA-Turner>
+* Armin Ronacher <armin.ronacher@active-4.com>
 * Daniel Neuhäuser <@DasIch>
-* Jon Waltman <@jonwaltman>
+* François Freitag <@francoisfreitag>
+* Georg Brandl <georg@python.org>
+* Jakob Lykke Andersen <@jakobandersen>
+* Jean-François Burnol <@jfbu>
 * Rob Ruana <@RobRuana>
 * Robert Lehmann <@lehmannro>
-* Roland Meister <@rolmei>
+* Stephen Finucane <@stephenfin>
+* Takayuki Shimizukawa <shimizukawa@gmail.com>
 * Takeshi Komiya <@tk0miya>
-* Jean-François Burnol <@jfbu>
-* Yoshiki Shibukawa <@shibu_jp>
-* Timotheus Kampik - <@TimKam>
-* Adam Turner - <@AA-Turner>
+* Timotheus Kampik <@TimKam>
+* Yoshiki Shibukawa <@shibukawa>
 
-Other contributors, listed alphabetically, are:
+Other contributors, listed alphabetically by forename, are:
```

Comment:
It's not just the forename, since two people with the same first name will be sorted by their second name. :)

Reply:
This isn't hypothetical either, there's multiple folks named Martin in this list. :)